pygdf/_gdf.py

```diff
@@ -367,3 +374,15 @@ def hash_partition(input_columns, key_indices, nparts, output_columns):
 
     offsets = list(offsets)
     return offsets
+
+
+def count_nonzero_mask(mask, size):
+    assert mask.size * mask_bitsize >= size
+    nnz = ffi.new('int*')
+    nnz[0] = 0
+    mask_ptr, addr = unwrap_mask(mask)
+    if addr != ffi.NULL:
+        libgdf.gdf_count_nonzero_mask(mask_ptr, size, nnz)
+    else:
+        nnz[0] = 0
```

Comment:
You shouldn't need this else clause because you set `nzz[0] = 0` above, no?

Reply:
True :)